digicampipe/io/generate_slow_containers.py

```diff
@@ -0,0 +1,203 @@
+import os
+import re
+import sys
+import pkg_resources
+from glob import glob
+
+from astropy.io import fits
+
+"""
+This script is looking in the digicampipe/tests/resources/slow/ folders and 
+looks for fits file.
+For each distinct type of file found (based on the filenames), a corresponding 
+container and a function to fill it is generated.
+Synchronisation of slow data with digicam data is made acording to the timestamp
+property of the slow data.
+In case you want to use slow data files with a different format than the one 
+currently used, empty the digicampipe/tests/resources/slow/ folder, place the 
+slow data you want to use in it an run this script.
+"""
+
+
+def unit_parser(column):
```

Comment:
Maybe we could make good use of `astropy.units` here. But this requires some major change in `digicampipe`

Reply:
I do use astropy.units (see generate_container_class() ), that parser is just to return units compatible with astropy.units when the description in the fits file is not.

Reply:
ah yes it creates units via Field()... sorry